tensilelite/Tensile/Source/lib/include/Tensile/SolutionLibrary.hpp

```diff
@@ -139,6 +139,12 @@ namespace Tensile
             return std::shared_ptr<MySolution>();
         }
 
+        virtual std::shared_ptr<MySolution> getSolutionByIndex(Hardware const&  hardware, const int index) const
+        {
+            throw std::runtime_error("[getSolutionByIndex] You should not reach here.");
```

Comment:
Should it throw error like above function or simply return shared_ptr<MySolution>()